crates/transaction-pool/src/error.rs

```diff
@@ -320,6 +326,11 @@ impl InvalidPoolTransactionError {
                         // this is a malformed transaction and should not be sent over the network
                         true
                     }
+                    Eip4844PoolTransactionError::UnexpectedEip4844SidecarAfterOsaka |
+                    Eip4844PoolTransactionError::UnexpectedEip7594SidecarBeforeOsaka => {
+                        // TODO: reconsider
+                        false
```

Comment:
I think this is appropriate, we don't want to penalize peers on the fork boundary

Reply:
but we do want to penalize peers far before and after the fork boundary